src/styles/GlobalStyle.js

```diff
@@ -1,5 +1,5 @@
 import { createGlobalStyle } from 'styled-components'
-import grasBackground from '../assets/gras-background.jpg'
+import GrasBackground from '../assets/gras-background.jpg'
```

Comment:
If you only use one background image, I would change the filename to "background.jpg".